themes/default/layouts/partials/header.html

```diff
@@ -109,31 +109,178 @@
+                                    </div>
+                                    <div class="header-nav-drop-down-sub-heading ">
+                                        Learn about the open source Pulumi Platform and managed Pulumi Service
+                                    </div>
+                                </div>
+                                <div class="header-nav-two-col-drop-down">
+                                    <li>
+                                        <div class="list-title">
+                                            <a href="{{ relref . "/product" }}">
+                                                <i class="fas fa-folder"></i>
+                                                Pulumi Overview
+                                                <div class="list-sub-title">
+                                                    Open source, infrastructure as code for Node.js, Python, Go, .NET, Java, and YAML
+                                                </div>
+                                            </a>
+                                        </div>
+                                    </li>
+                                    <li>
+                                        <div class="list-title">
+                                            <a href="{{ relref . "/product/pulumi-service" }}">
+                                                <i class="fas fa-user-cog"></i>
+                                                Pulumi Service
+                                                <div class="list-sub-title">
+                                                    Managed service for adopting Pulumi open source at scale.
+                                                    Use SaaS or self-host
+                                                </div>
+                                            </a>
+                                        </div>
+                                    </li>
+                                    <li>
+                                        <div class="list-title">
+                                            <a href="{{ relref . "/docs/reference/pulumi-sdk" }}">
+                                                <i class="fas fa-database"></i>
+                                                SDK & Engine
+                                                <div class="list-sub-title">
+                                                    Provision and manage infrastructure on any cloud
+                                                    interactively with a CLI
+                                                </div>
+                                            </a>
+                                        </div>
+                                    </li>
+                                    <li>
+                                        <div class="list-title">
+                                            <a href="{{ relref . "/docs/intro/pulumi-service/ci-cd-integration-assistant" }}">
```

Comment:
Is http://pulumi-hugo-origin-pr-1477-1872f80c.s3-website.us-west-2.amazonaws.com/docs/guides/continuous-delivery/ a better target?

Reply:
I agree since the one that is there now is specific to configuring the CI/CD assistant rather than linking to the general CI/CD page.